portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,155 @@
+                <p>In college, I have become an active leader in the Honors Academy in the School of Engineering. I help interview perspective students and run panels to answer FAQ's. I love to get involved, and I'm glad Google SPS will be a step in the right direction. </p>
+             </div>
+
+             <div class="card">
+                <h2>What I enjoy to photograph</h2>
+                <p>I love to photograph anything that can trace back to a memory. Here are some pictures I have taken. </p>
+                <div class="row"> 
+
+                    <div class="column">
+                        <img src="/images/1.png" style="width:100%">
+                        <img src="/images/2.png" style="width:100%">
+                        <img src="/images/4.png" style="width:100%">
+                        <img src="/images/9.png" style="width:100%">
+                    </div>
+
+                    <div class="column">
+                        <img src="/images/3.png" style="width:100%">
+                        <img src="/images/8.png" style="width:100%">
+                    </div>  
+
+                    <div class="column">
+                        <img src="/images/5.png" style="width:100%">
+                        <img src="/images/6.png" style="width:100%">
+                        <img src="/images/7.png" style="width:100%">
+                    </div>
+
+                </div>
+
+                </div>
+        </div>
+
+        <div class="rightcolumn">
+
+            <div class="card">
+                <button class="button" onclick="addRandomGreeting()">Click for a fun fact about me</button>
+                <br>
+                <br>
+                <div id="greeting-container"></div>
+            </div>
+
+            <div class="card">
+                <h3>Past Projects</h3>
+                <ul>
+                    <li> <a href="http://clarity-support.appspot.com/">Clarity</a> a site for self esteem.</li>
+                    <li>Automatic Fire Extinguisher using Arduino</li>
+                    <li>Staying Awake with Computer Vision - A preventative measure against drowsy driving</li>
+                </ul>
+      
+            </div>
+
+            <div class="card">
+                <h3>Hidden Talents</h3>
+                <ul>
+                    <li>I can speak Mandarin, Gujarati, and of course, English</li>
+                    <li>I can play the piano and percussion</li>
+                    <li>Not really a talent, but I am left-handed</li>
+                </ul>
+            </div>
+
+            <div class="card">
+                <h3>Photo Gallery</h3>
+                <div class="container">
+
+                    <div class="mySlides">
+                        <div class="numbertext">1 / </div>
+                        <img src="/images/a.png" style="width:100%">
+                    </div>
+
+                    <div class="mySlides">
+                        <div class="numbertext">3 / 6</div>
```

Comment:
Where's 2/6? Also missing b.png below

Reply:
the b.png image wasn't uploading for some reason so I just got rid of it.